.github/workflows/ci.yml

```diff
@@ -76,7 +84,16 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: '**/build'
-          key: ${{ runner.os }}-build-${{ github.sha }}
+          key: ${{ runner.os }}-build-${{ github.sha }}      
+
+      - name: Install dependencies
```

Comment:
I don't these steps are needed, as they are done in the previous job, `build`, that this one needs. Same for the `corepack`step.